src/OpenLoco/src/Ui/ProgressBar.cpp

```diff
@@ -14,16 +14,8 @@ namespace OpenLoco::Ui::ProgressBar
 
     void begin(std::string_view caption)
     {
-        if (isInitialised())
-        {
-            _isInternalWindow = true;
-            Windows::ProgressBar::open(caption);
-        }
-        else
-        {
-            // Used to be the native progress bar
-            assert(false);
-        }
+        _isInternalWindow = true;
```

Comment:
if we are doing this might as well remove the other `if (_isInternalWindow)` and then remove the variable as well